Cargo.toml

```diff
@@ -35,3 +35,4 @@ wasmtime                        = "0.30"
 wasmtime-wasi                   = "0.30"
 wasmtime-cache                  = "0.30"
 wat                             = "1.0.37"
+chrono = "0.4.19"
```

Comment:
nit: arrange TOML entries to the right.

Reply:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO